.eslintrc.ts

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    "google",
+    "plugin:vue/vue3-essential",
+    "plugin:prettier/recommended",
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+  ],
```

Comment:
Consider adding the following:
```
    'plugin:vue/base',
    'plugin:vue/essential',
    'plugin:vue/recommended',
    'plugin:vue/strongly-recommended',
    'plugin:vue/vue3-recommended',
    'plugin:vue/vue3-strongly-recommended',
    'plugin:vuetify/base',
    'plugin:vuetify/recommended',
    '@vue/standard',
    '@vue/typescript'
```

Some of these need extra packages. Maybe these?
```
    "@typescript-eslint/eslint-plugin": "^5.59.2",
    "@typescript-eslint/parser": "^5.59.2"
    "@vue/eslint-config-standard": "^4.0.0",
    "@vue/eslint-config-typescript": "^9.1.0",
    "eslint": "~7.32.0",
    "eslint-plugin-vue": "^8.7.1",
    "eslint-plugin-vuetify": "^1.1.0",
```

Reply:
(Ignore the Vuetify items if you're not using Vuetify.)

Reply:
Please do not resolve the conversations that I started.

I leave them unresolved as a marker to re-review just the changed code (otherwise I have to re-review all the code).

Reply:
This conversation can stay resolved as I have verified the changes.